client/src/components/ui/contextdrawer/PersonContextGlobal.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+
+    import { Events, PersonPayload, IconSize } from '../../types.ts';
+    import ContextTemplate from '../contextmenu/ContextTemplate.svelte';
+    import ContextElement from '../contextmenu/ContextElement.svelte';
+    import ContextDivider from '../contextmenu/ContextDivider.svelte';
+
+    import Edit from '../../icons/Edit.svelte';
+    import PersonDelete from '../../icons/PersonDelete.svelte';
+
+    const dispatch = createEventDispatcher();
+    export let show = false;
```

Comment:
This should work around the lint errors.
```suggestion
    export let show = false as boolean;
```